src/Documentation/Grain-Versioning/Compatible-grains.md

```diff
@@ -0,0 +1,22 @@
+# Compatible grains
+
+In this document will be described the notion of "backward compatible" and "fully compatible".
+
+## Backward compatible
```

Comment:
This section is confusing, because it seems you are talking about what to have in mind to create versions that are back compat or fully back and forward compat, but it's not cleat that apart from these tips, there is a policy that can be configured and will actually be used. 
Also I would tie/link this with the `Version selector strategy` doc.